src/modules/hip/kernel/roi_conversion.hpp

```diff
@@ -0,0 +1,29 @@
+#include <hip/hip_runtime.h>
+
+extern "C" __global__ void roi_converison_ltrb_to_xywh(int *roiTensorPtrSrc)
+{
+    int id_x = (hipBlockIdx_x * hipBlockDim_x + hipThreadIdx_x) * 4;
+
+    roiTensorPtrSrc[id_x + 2] -= (roiTensorPtrSrc[id_x] - 1);
```

Comment:
use int2 to read from the roiTensorPtrSrc instead of using +1

Reply:
Thanks! I have changed it now. Also, here roiTensorPtrSrc has 0/1/2/3, so I have used int4.